packages/TagPicker/src/TagPicker.tsx

```diff
@@ -297,7 +316,8 @@ const TagPicker: React.FunctionComponent<TagPickerProps> = ({
                 color={s.color}
                 icon={s.icon}
                 dismissible={!disabled}
-                appearance="neutral"
+                appearance={getBrand() === "workleap" ? "default" : "neutral"}
+                rounded={getBrand() === "workleap"}
```

Comment:
With version 1.2.0 of the `Tag` component, this declaration is no longer necessary, since by default they are round.